.gitignore

```diff
@@ -0,0 +1,2 @@
+configuration.yaml
```

Comment:
You should provide a `configuration-example.yaml` with dummy data and let your user know how to configure this app. Example: https://github.com/osu-mist/express-api-skeleton/blob/master/src/tests/integration/configuration-example.json

Reply:
done.